src/Event/Productions/SimilaritiesClient.php

```diff
@@ -64,7 +65,7 @@ public function excludePermanently(SimilarEventPair $pair)
         );
     }
 
-    public function nextSuggestion(\DateTime $dateFrom, int $size = 1, int $offset = 0): Suggestion
+    public function nextSuggestion(Date $dateFrom, int $size = 1, int $offset = 0): Suggestion
```

Comment:
We usually use `DateTimeImmutable`, but maybe in this case `Date` is better yes since we don't store the time info anyway.